dali/test/python/test_dali_variable_batch_size.py

```diff
@@ -263,6 +329,15 @@ def test_ops_image_default_args():
     (fn.rotate, {'angle': 25}),
     (fn.transpose, {'perm': [2, 0, 1]}),
     (fn.warp_affine, {'matrix': (.1, .9, 10, .8, -.2, -20)}),
+    (fn.expand_dims, {'axes': 1, 'new_axis_names': "Z"}),
+    (fn.grid_mask, {'tile': 51, 'ratio': 0.38158387, 'angle': 2.6810782}),
+    (numba_function, {'run_fn': numba_set_all_values_to_255_batch, 'out_types': [types.UINT8],
```

Comment:
Apologises for nitpicking, but maybe keeping the alphabetic order of `ops_image_custom_args` would look nice :)

Reply:
Done